app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -208,6 +209,8 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
     private var mFilePathCallback: ValueCallback<Array<Uri>>? = null
     private var isConnected = false
     private var isShowingError = false
+    private var isPageFinished = false
+    private var pageError = 0 // 0 - OK, <0 - Connection Error, >=400 - HTTP Error
```

Comment:
What purpose does this have? It is not used in the UI? I can only see it in a log statement when reloading at which point it is no longer relevant because there is reloading 🙃

Reply:
Used in `reloadPage`. It is possible to show more information about the error to the user.

Reply:
But you're not showing anything. For debugging with http status codes etc. you may as well use [Chrome remote debugging](http://companion.home-assistant.io/docs/integrations/android-webview/#remote-debugging), the app is never going to match that level of detail.

Reply:
Currently used for status in WebView.
It is now possible to show the tip.
`pageError < 0`:
> Check your network connection.

`pageError >= 400` (Android 6.0):
> Check the server configuration.